Supervision Work/src/uk/ac/cam/ap886/oopjava/supervisionProjects/Hangman/CLI.java

```diff
@@ -0,0 +1,209 @@
+package uk.ac.cam.ap886.oopjava.supervisionProjects.Hangman;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStreamReader;
+
+public class CLI implements UserInterface {
+	
+	
+	Game game;
+	BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
+	
+	
+	//----------------
+	
+	public CLI (Game newGame){
+		game = newGame;
+	}
+	
+	
+	
+	private void prn(String arg){
+		System.out.println(arg);
+	}
+	
+	private String scn(){
+		try {
+			return br.readLine().trim();
+		} catch (IOException e) {
+			e.printStackTrace();
+			return null;
+		}
+	}
+	
+	private char getch(){
+		try{
+			return br.readLine().charAt(0);
+		}catch(IOException e){
+			e.printStackTrace();
+			return '\0';
```

Comment:
I think it would be better to just bubble the exception up. It might want to be handled specially.